conda_environment.yml

```diff
@@ -2,5 +2,5 @@ channels:
     - conda-forge
 dependencies:
     - python>=3.6
-    - iris>=3.1
-    - cartopy>=0.20
\ No newline at end of file
+    - iris>=3.1,!=3.2.0.post0
```

Comment:
I don't really understand the decision for this yet: In #53 you mention that the actual errors leave when you install Iris version 3.1. Why not change this to `iris==3.1` or `iris>=3.1,<=3.2`, then?

Reply:
Especially if Iris drops support for Python 3.7 from now on, I think we should start thinking if we just keep it at the current version or require 3.8 or higher.

Reply:
But still, the tests are failing with Iris 3.2 (without the `.post0` prefix). So maybe use `!=3.2` instead of `!=3.2.0.post0`?

Reply:
Alright, I see that there are actually two buggy package candidates on conda-forge, I haven't seen this earlier (and it was working for me).
So `!=3.2.0.post0` is not enough, but what about `!=3.2.0`, which is relaxing the excluding rule as little as possible? It is also what the ESMValTool people seem to agree on.

Reply:
... should have read the whole discussion over there. Both `!=3.2.0.post0` and `!=3.2.0` need to be pinned in that case.

Reply:
Yes, that sounds a lot better! Also sounds pretty much like what has been discussed in the other thread :)